### PR TITLE
Get repo ready for Android Studio Arctic Fox

### PR DESCRIPTION
**Pull request by chander-stripe**
Notably, Arctic Fox mandates Java 11, instead of Java 8.

### CODE REVIEW

File: Example/javaapp/build.gradle [hunk 1]
```diff
@@ -13,8 +13,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```
Comment (mshafrir-stripe) [inline Example/javaapp/build.gradle:17]: If a library project targets Java 11, can an app that targets an earlier version include it as a dependency?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ For information on migrating from earlier beta versions of the Android SDK, see
 
 # Requirements
 
-The Stripe Terminal Android SDK is compatible with apps supporting Android API level 24 and above. Apps can be written using Kotlin or [Java 8](https://developer.android.com/studio/write/java8-support).
+The Stripe Terminal Android SDK is compatible with apps supporting Android API level 24 and above. Apps can be written using Kotlin or [Java 11](https://developer.android.com/studio/write/java8-support).
```
Comment (danwaters-stripe) [inline README.md:7]: I don't think we want to update this yet, since any releases for which this is true are still in beta. (I think we should probably revert the change to say API level 24 too)
